### PR TITLE
Allow some WDL functions to be called in a workflow context

### PR DESCRIPTION
**Pull request by scottfrazer**
This is a problem that Dave Shiga and Jackie both ran into in the past week.

https://broadinstitute.atlassian.net/browse/DSDEEPB-1544

### CODE REVIEW

**Comment by mcovarr** [comment]
There are two failures in WorkflowManagerActorSpec, but I believe they are both due to the asynchronicity of setting status that should be fixed (inelegantly) by #224.

File: src/main/scala/cromwell/engine/backend/jes/JesEngineFunctions.scala [hunk 1]
```diff
@@ -62,4 +49,29 @@ case class JesEngineFunctions(jesBackendCall: JesBackendCall) extends WdlStandar
...
+ * Implementation of WDL standard library functions for the JES backend.
+ */
+class JesEngineFunctions(jesBackendCall: JesBackendCall) extends JesEngineFunctionsWithoutCallContext(jesBackendCall.jesConnection.storage) {
+
+  override def readFromPath(value: String): String = {
+    val gcsPath = GoogleCloudStoragePath.parse(value) match {
+      case Success(path) => path
+      case Failure(ex) => GoogleCloudStoragePath(jesBackendCall.callDir + s"/$value")
+    }
+    jesBackendCall.jesConnection.storage.slurpFile(gcsPath)
```
Comment (geoffjentry) [inline src/main/scala/cromwell/engine/backend/jes/JesEngineFunctions.scala]: \n

**Comment by geoffjentry** [comment]
:+1:

**Comment by mcovarr** [comment]
:+1:

**Comment by Horneth** [comment]
I :+1: too since I'm assigned :)

**Comment by mcovarr** [comment]
This should be ready to merge with a rebase.

**Comment by scottfrazer** [comment]
I've been delaying the rebase because I need `jes_robustness` merged first.

**Pull request merged**